regolith/helpers/u_milestonehelper.py

```diff
@@ -0,0 +1,138 @@
+"""Helper for updating milestones to the projecta collection.
+    It can update the status and due date of a projectum.
+    It can add a new milestone to the projecta collection.
+"""
+import datetime as dt
+from nameparser import HumanName
+import dateutil.parser as date_parser
+import sys
+import uuid
+
+from regolith.helpers.basehelper import DbHelperBase
+from regolith.fsclient import _id_key
+from regolith.tools import all_docs_from_collection
+
+
+TARGET_COLL = "projecta"
+
+def subparser(subpi):
+    subpi.add_argument("projectum_id", help="the id of the projectum")
+    subpi.add_argument("--number",
+                        help="number of the milestone to update from numbered list")
+    subpi.add_argument("--due_date",
+                       help="new due date of the milestone"
+                            "required to add a new milestone")
+    subpi.add_argument("-s", "--status",
```

Comment:
see below, you could maybe take a single letter that stands for each allowed entry?

Reply:
hi professor, can you confirm if is that what you meant? 
I did this, and added a dictionary with those keys in the code

Reply:
yes.  Not sure if it is good, but that is what I was suggesting.